File: abc/abc205/a.cs
// https://atcoder.jp/contests/abc205/submissions/46503058

using System;

public class A
{
  static void Main()
  {
    string[] t = Console.ReadLine().Split(" ");
    int a = int.Parse(t[0]);
    double b = double.Parse(t[1]);

    Console.WriteLine(b / 100 * a);
  }
}

File: abc/abc205/a.rs
// https://atcoder.jp/contests/abc205/submissions/47813159

use proconio::input;

fn max(a: u16, b: u16) -> u16 {
    if a > b {a} else {b}
}

fn main() {
    input! {
        a: f64,
        b: f64,
    }

    print!("{}", a / 100.0 * b);
}

